### PR TITLE
gisto 0.3.0 [license type added]

### PR DESCRIPTION
**Pull request by ryanpcmcquen**

### CODE REVIEW

File: Casks/gisto.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  version '0.3.0'
+  sha256 '803a15125b69486012f72a42dd20f7a63b51dc2d445082303cb3b96cdb622667'
 
-  url "http://download.gistoapp.com/Gisto-#{version}-OSX_x86.dmg"
+  url "http://download.gistoapp.com/Gisto-#{version}-OSX_x86_64.dmg"
   appcast 'http://www.gistoapp.com/GistoAppCast.xml'
   name 'Gisto'
   homepage 'http://www.gistoapp.com/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license MIT
```
Comment (fanquake) [inline Casks/gisto.rb]: This needs to be :MIT

**Comment by ryanpcmcquen** [comment]
@fanquake thanks for the heads up! Fixed now.  :-)

**Comment by fanquake** [comment]
@ryanpcmcquen No worries. Can you just squash your commits and well get this merged.

**Comment by ryanpcmcquen** [comment]
@fanquake squashed it! Thanks again.  :-)

**Pull request merged**